docs/getting-started/nullblk.md

```diff
@@ -81,9 +81,17 @@ command to configure the emulated disk.
 	zone_size=64 \
 ```
 
+In this example, the arguments mean the following:
+
+   1. ``nr_devices=1`` means that the number of specified devices is one (1). 
+   2. ``zoned=1`` means that one (1) zoned block device is created.
```

Comment:
...means that all devices created will be zoned devices...

zoned=1 enables "zone mode" for any device that the modprobe command will create.

Reply:
I've made this change.